overview/devmode.nix

```diff
@@ -0,0 +1,19 @@
+let
+  default = import ../. { };
+in
+import ./. {
+  inherit (default)
+    lib
+    lib'
+    system
+    projects
+    ;
+  pkgs = with default; pkgs // ngipkgs;
+  nixpkgs = default.sources.nixpkgs;
+  options = default.optionsDoc.optionsNix;
+  self = {
```

Comment:
Oof. (not actionable)

Reply:
> We need to eventually figure out how to do the linking of paths properly

I can't reproduce the message you've posted, but the fonts should work with https://github.com/NixOS/nixpkgs/pull/404401

> Oof. (not actionable)

Do you mean how this is called or the `self` fallback?

Reply:
Yes, this whole `self` stuff... Although this probably can be resolved with an equivalent of flake-parts.

Reply:
> I can't reproduce the message you've posted, but the fonts should work with https://github.com/NixOS/nixpkgs/pull/404401

How's that? The fonts aren't symlinked. It's that the CSS refers to them by absolute path